README.md

```diff
@@ -34,18 +35,21 @@ import { Client } from 'ff-nodejs-server-sdk';
 ### Initialize
 
 This is the most simple way to initialize SDK using only a server type key
+
 ```
```

Comment:
Try to add the language type here so GitHub can be more intelligent about its highlighting
```suggestion
```typescript
```